dist/live.html

```diff
@@ -5,119 +5,104 @@
-        margin-bottom: 18px;
-        white-space: nowrap
-      }
-
-      span {
-        font-size: inherit;
-        line-height: inherit;
-        margin-top: inherit;
-        margin-bottom: inherit;
-      }
-
-      span.location-emoji {
-          margin-left: -2px;
-          margin-right: -5px;
-      }
-
-      p.highlight {
-        background: #CE481C;
-        color: white;
-        padding-top: 6px;
-        padding-bottom: 2px;
-        /* to compensate for the padding */
-        margin-top: 12px;
-        margin-bottom: 16px;
-      }
-
-      p.done, p.done span {
-        opacity: 0.5;
-        line-height: 8px;
-      }
-
-      #title-date { font-size: 20px }
-
-      #timeline { padding-right: 300px }
-
-      #twitter-container {
-        position: fixed;
-        top: 0px;
-        right: -1px;
-        width: 300px;
-        height: 100%;
-        border-left: 1px solid rgba(15, 70, 100, .12);
-      }
-
-      @media screen and (max-width: 1000px) {
-        #twitter-container { display: none }
-
-        body {
-          padding-top: 10px;
-          padding-bottom: 10px;
-        }
-
-        h1, h2, p { padding-left: 10px }
-
-        p, p.done {
-          white-space: normal;
-          line-height: 16px;
-          font-size: 16px;
-        }
-      }
-    </style>
   </head>
   <body>
-    <h1># HACKUPC Live ヽ(◕‿◕)ﾉ <span id="title-date"></span></h1>
-    <div id="timetable"></div>
-
-    <div id="twitter-container">
-        <a class="twitter-timeline" height="100%" href="https://twitter.com/search?q=%28hackupc%20OR%20from%3Ahackupc%29%20AND%20-filter%3Aretweets" data-widget-id="700696875549102080" data-link-color="#CE481C">Tweets about (hackupc OR from:hackupc) AND -filter:retweets</a>
-        <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
+    <div></div>
+    <div id="app">
+        <div class="navbar col-sm-12" id="livenav">
+            <div class="container-fluid">
+                <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                    <ul class="nav navbar-nav col-md-12">
+                        <li class="active col-sm-2 col-sm-offset-1"><a href="#">Welcome</a></li>
+                        <li class="active col-sm-2"><a href="#">Maps</a></li>
+                        <li class="active col-sm-2"><a href="#">LIVE</a></li>
+                        <li class="active col-sm-2"><a href="#">The Game</a></li>
+                        <li class="active col-sm-2"><a href="#">MLH Hardware Lab</a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+
+        <h1># HACKUPC Live ヽ(◕‿◕)ﾉ <span id="title-date"></span> <span v-on:click="options" class="fa fa-cog"></span></h1>
```

Comment:
Add small countdown of remaining hacking hours 

- Before the event -> 36h
- During the Event -> countdown
- After the event -> Link to submit you hack!